src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,13 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        final int ballsQuantity = 3;
+        Ball[] lotteryBalls = new Ball[ballsQuantity];
```

Comment:
Do we really need to store the balls in the array or we can just print info in the loop when we creating the ball?

Reply:
Fixed